### PR TITLE
Pablo`s branch

### PR DESCRIPTION
**Pull request by PavelKostyrko**
I have completed first and second tasks and giving a pull request.

### CODE REVIEW

File: FitnessApp.Data/Configurations/ProductNutrientConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Threading.Tasks;
+
+namespace FitnessApp.Data.Configurations
+{
+    internal class ProductNutrientConfiguration : IEntityTypeConfiguration<ProductNutrientDb>
+    {
+        public void Configure(EntityTypeBuilder<ProductNutrientDb> builder)
+        {
+            builder.ToTable("ProductsNutrients").HasKey(t => t.Id); // configure table name and set primary key
+            builder.Property(_ => _.Id).ValueGeneratedOnAdd(); // auto creating id when entity is added
```
Comment (Eugene-Patysh) [inline FitnessApp.Data/Configurations/ProductNutrientConfiguration.cs:17]: where are relations with Product, Nutrient and TreatingType?

File: FitnessApp.Data/ProductContext.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        public ProductContext(DbContextOptions<ProductContext> options)
+            : base(options)
+        {
+            Database.EnsureCreated();
+        }
+        public DbSet<ProductNutrientDb> ProductNutrients { get; set; }
+        public DbSet<TreatingTypeDb> TreatingTypes { get; set; }
+        public DbSet<ProductDb> Products { get; set; }
+        public DbSet<ProductCategoryDb> ProductCategories { get; set; }
+        public DbSet<ProductSubCategoryDb> ProductSubCategorv { get; set; }
```
Comment (Eugene-Patysh) [inline FitnessApp.Data/ProductContext.cs]: incorrect name

File: FitnessApp.Data/Configurations/ProductNutrientConfiguration.cs (hunk 1 above)
Comment (PavelKostyrko) [reply FitnessApp.Data/Configurations/ProductNutrientConfiguration.cs:17]: Done. But, I think it`s not correct.

File: FitnessApp.Data/ProductContext.cs (hunk 2 above)
Comment (PavelKostyrko) [reply FitnessApp.Data/ProductContext.cs]: Done.

**Pull request merged**